distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -135,8 +135,6 @@ public void startManagedService(String servicePid, Map<String, Object> propertie
     @Override
     public void stopManagedService(String servicePid) throws IOException {
         Configuration sourceConfig = adminConfig.getConfiguration(servicePid, null);
-        ServiceManagerImpl.ServiceConfigurationListener listener =
```

Comment:
@lessarderic  I talked to tracy who said that he didn't think the listener was being used. Are you sure I would need to unsubscribe it? I figured if it wasn't being used and it was created here, it would be best if we just got rid of the listener. Thoughts?